File: src/main/kotlin/org/aresclient/ares/api/render/TextColor.java
package org.aresclient.ares.api.render;

public enum TextColor {
    BLACK('0'),
    DARK_BLUE('1'),
    DARK_GREEN('2'),
    DARK_AQUA('3'),
    DARK_RED('4'),
    DARK_PURPLE('5'),
    GOLD('6'),
    GRAY('7'),
    DARK_GRAY('8'),
    BLUE('9'),
    GREEN('a'),
    AQUA('b'),
    RED('c'),
    LIGHT_PURPLE('d'),
    YELLOW('e'),
    WHITE('f');

    private final String string;

    TextColor(char key) {
        this.string = "\u00a7" + key;
    }

    @Override
    public String toString() {
        return string;
    }
}

File: src/main/kotlin/org/aresclient/ares/api/nrender/TextColor.kt
package org.aresclient.ares.api.nrender

enum class TextColor(key: Char) {
    BLACK('0'),
    DARK_BLUE('1'),
    DARK_GREEN('2'),
    DARK_AQUA('3'),
    DARK_RED('4'),
    DARK_PURPLE('5'),
    GOLD('6'),
    GRAY('7'),
    DARK_GRAY('8'),
    BLUE('9'),
    GREEN('a'),
    AQUA('b'),
    RED('c'),
    LIGHT_PURPLE('d'),
    YELLOW('e'),
    WHITE('f');

    private val string = "\u00a7" + key

    override fun toString(): String {
        return string
    }
}
